lib/cext/ABI_version.txt

```diff
@@ -1 +1 @@
-9
+10
```

Comment:
I don't think you need to change this. The header allowed linking against `rb_strerr_new`, it just couldn't resolve the function at runtime since the definition wasn't available. As far as I can tell, no existing extensions will need recompilation when this change lands.

Reply:
ABI_version.txt should be bumped because `have_func('rb_syserr_new')` would be false before this PR and true after.
`have_func` tries to link, it's not only checking the header, and so it does notice the case of declared in header but no implementation.

Reply:
> As far as I can tell, no existing extensions will need recompilation when this change lands.

Which is probably true if they called `rb_strerr_new()` unconditionally, but wouldn't if they do things like `have_func('rb_syserr_new')` + `#ifdef HAVE_RB_SYSERR_NEW`.